go/host/db/batches.go

```diff
@@ -132,25 +132,30 @@ func (db *DB) GetBatchBySequenceNumber(sequenceNumber *big.Int) (*common.ExtBatc
 	return db.GetBatch(*batchHash)
 }
 
-// GetBatchListing returns BatchListingResponse given a pagination
+// GetBatchListing returns latest BatchListingResponse given a pagination. For example, page 0, size 10 will return the
+// latest 10 batches.
```

Comment:
Would it make sense to make all listings return in reverse order and having a comment making that super clear ?

Reply:
Yeah good point, I'll see if I can make this more general, the rollups should behave the same.